runners/pvc_autoresizer_test.go

```diff
@@ -99,39 +159,69 @@ var _ = Describe("test resizer", func() {
 			volumeMode := corev1.PersistentVolumeFilesystem
 
 			testCases := []struct {
-				description   string
-				pvcSizeGi     int64
-				expectSizeGi  int64
-				threshold     string
-				availableByte int64
+				description        string
+				pvcSizeGi          int64
+				expectSizeGi       int64
+				threshold          string
+				availableByte      int64
+				capacityInodeSize  int64
+				availableInodeSize int64
+				inodesThreshold    string
 			}{
 				{
-					description:   "Should resize(absolute value)",
-					pvcSizeGi:     10,
-					expectSizeGi:  11,
-					threshold:     "5Gi",
-					availableByte: 5<<30 - 1,
+					description:        "Should resize(absolute value)",
+					pvcSizeGi:          10,
+					expectSizeGi:       11,
+					threshold:          "5Gi",
+					availableByte:      5<<30 - 1,
+					availableInodeSize: 100,
+					capacityInodeSize:  100,
+				},
+				{
+					description:        "Should not resize(absolute value)",
+					pvcSizeGi:          10,
+					expectSizeGi:       10,
+					threshold:          "5Gi",
+					availableByte:      5 << 30,
+					availableInodeSize: 100,
+					capacityInodeSize:  100,
 				},
 				{
-					description:   "Should not resize(absolute value)",
-					pvcSizeGi:     10,
-					expectSizeGi:  10,
-					threshold:     "5Gi",
-					availableByte: 5 << 30,
+					description:        "Should resize(%)",
+					pvcSizeGi:          10,
+					expectSizeGi:       11,
+					threshold:          "50%",
+					availableByte:      5<<30 - 1,
+					availableInodeSize: 100,
+					capacityInodeSize:  100,
 				},
 				{
-					description:   "Should resize(%)",
-					pvcSizeGi:     10,
-					expectSizeGi:  11,
-					threshold:     "50%",
-					availableByte: 5<<30 - 1,
+					description:        "Should not resize(%)",
+					pvcSizeGi:          10,
+					expectSizeGi:       10,
+					threshold:          "50%",
+					availableByte:      5 << 30,
+					availableInodeSize: 100,
+					capacityInodeSize:  100,
 				},
 				{
-					description:   "Should not resize(%)",
-					pvcSizeGi:     10,
-					expectSizeGi:  10,
-					threshold:     "50%",
-					availableByte: 5 << 30,
+					description:        "Should resize(inode)",
+					pvcSizeGi:          10,
+					expectSizeGi:       11,
+					threshold:          "50%",
+					availableByte:      5 << 30,
+					availableInodeSize: 9,
+					capacityInodeSize:  100,
+				},
+				{
+					description:        "Should resize(inode with annotation)",
+					pvcSizeGi:          10,
+					expectSizeGi:       11,
+					threshold:          "50%",
+					availableByte:      5 << 30,
+					availableInodeSize: 49,
+					capacityInodeSize:  100,
+					inodesThreshold:    "50%",
```

Comment:
Could you please add a test case when `inodesThreshold` is set to 0. And confirm that resize caused by inode does not work in this combination.

Reply:
added.
https://github.com/topolvm/pvc-autoresizer/pull/65/commits/ba890c276c635401ca27133583fc5f27830086e1

Reply:
@bells17 
Excuse me. Could you please add one more test case of `availableInodeSize` and `capacityInodeSize` are set to 0.
I believe that the behavior of `GetMetrics` method is the same whether the metrics are empty and are 0. Aren't you?

Reply:
Sorry, I added a testcase for your comment.
https://github.com/topolvm/pvc-autoresizer/commit/b69df085e47a0bbe0ab1752552bbfa1a62a6f435